### PR TITLE
Add functions to get inputs to hash

### PR DESCRIPTION
**Pull request by samanehsan**
### Purpose/Changes
Refactor pipeline tools functions to incorporate getting fastq file hashes for labeling workflows with a hash of its bundle-specific inputs. This will keep all of the pipeline logic inside of pipeline-tools.

### Review Instructions
Any feedback on approach here would be appreciated, since I did a bit of refactoring of the get-input-tsv functions!

### CODE REVIEW

File: pipeline_tools/pipelines/optimus/optimus.py [hunk 1]
```diff
@@ -18,6 +18,45 @@
...
+        r2_hashes = metadata_utils.get_hashes_from_file_manifest(
+            lane_to_fastqs[lane]['read2']
+        )
+        file_hashes = '{0}{1}'.format(r1_hashes, r2_hashes)
+        if lane_to_fastqs[lane].get('index1'):
+            i1_hashes = metadata_utils.get_hashes_from_file_manifest(
+                lane_to_fastqs[lane]['index1']
+            )
+            file_hashes += i1_hashes
+    # This order must be maintained to compare input hashes between different Optimus workflows
```
Comment (rexwangcc) [inline pipeline_tools/pipelines/optimus/optimus.py]:
```suggestion
    # NOTE: This order MUST be maintained to compare input hashes between different Optimus workflows!
```

Comment (rexwangcc) [reply pipeline_tools/pipelines/optimus/optimus.py]: (Just to make it more vigilant)

File: pipeline_tools/pipelines/optimus/optimus.py [hunk 2]
```diff
@@ -18,6 +18,45 @@
...
+    sorted_lanes = sorted(lane_to_fastqs.keys(), key=int)
+    file_hashes = ''
+    for lane in sorted_lanes:
+        r1_hashes = metadata_utils.get_hashes_from_file_manifest(
+            lane_to_fastqs[lane]['read1']
+        )
+        r2_hashes = metadata_utils.get_hashes_from_file_manifest(
+            lane_to_fastqs[lane]['read2']
+        )
+        file_hashes = '{0}{1}'.format(r1_hashes, r2_hashes)
```
Comment (rexwangcc) [inline pipeline_tools/pipelines/optimus/optimus.py]:
```suggestion
        file_hashes = f'{r1_hashes}{r2_hashes}'
```

**Comment by samanehsan** [comment]
retest this please

**Comment by samanehsan** [comment]
Going to merge this into the base branch `se-get-file-hashes`

**Pull request merged**